main.js

```diff
@@ -145,5 +150,18 @@ function assignCategory() {
 
 function createDataModel() {
   assignCategory();
-  currentActivity = new Activity(categorySelection, accomplishInput.value, minutesInput.value, secondsInput.value,)
+  currentActivity = new Activity(categorySelection, accomplishInput.value, minutesInput.value, secondsInput.value)
```

Comment:
This function looks great! We can optimize this later to include the countdown functionality.